spec/hello_spec.rb

```diff
@@ -1,13 +1,14 @@
 require "spec_helper"
 
-describe "#hello_t" do 
+describe "#hello_t" do
   let (:names) { ["Tim", "Tom", "Jim"] }
-  it "only passes names that start with 'T' to the block" do 
+
+  it "only passes names that start with 'T' to the block" do
     expect{hello_t(names){|name| puts "Hi, #{name}" if name.start_with?("T") }}.to output("Hi, Tim\nHi, Tom\n").to_stdout
   end
 
-  it "returns an array with only names that start with 'T'" do
-    expect(hello_t(names) {|name| puts "Hi, #{name}" if name.start_with?('T') }).to eq(names)
+  it "returns the original array" do
+    expect(hello_t(names) {|name| puts "Hi, #{name}" if name.start_with?("T") }).to eq(names)
   end
 
   it "is case insensitive" do
```

Comment:
The point of this test isn't really to test fo case insensitivity...It's actually to make sure the student isn't just hardcoding the logic to be `if name.starts_with("T")`. We should rename the test to reveal that.